batch/test/test_batch.py

```diff
@@ -1110,7 +1110,8 @@ def test_verify_no_access_to_metadata_server(client: BatchClient):
     status = j.wait()
     assert status['state'] == 'Failed', str((status, b.debug_info()))
     job_log = j.log()
-    assert "Connection timed out" in job_log['main'], str((job_log, b.debug_info()))
+    # Curl output 'Connection timed out' changed to 'Connection timeout' with ubuntu image update
```

Comment:
This comment probably belongs in the PR, not the code.  The code is static and "ubuntu update" won't make senes over time.